kong/plugins/opentelemetry/schema.lua

```diff
@@ -35,7 +35,8 @@ return {
     { config = {
       type = "record",
       fields = {
-        { endpoint = typedefs.url { required = true, referenceable = true } }, -- OTLP/HTTP
+        { traces_endpoint = typedefs.url { referenceable = true } }, -- OTLP/HTTP
+        { logs_endpoint = typedefs.url { referenceable = true } },
         { headers = { description = "The custom headers to be added in the HTTP request sent to the OTLP server. This setting is useful for adding the authentication headers (token) for the APM backend.", type = "map",
```

Comment:
> OTLP/HTTP Request
> Telemetry data is sent via HTTP POST request. The body of the POST request is a payload either in binary-encoded Protobuf format or in JSON-encoded Protobuf format.
> 
> The default URL path for requests that carry trace data is /v1/traces (for example the full URL when connecting to “example.com” server will be https://example.com/v1/traces). The request body is a Protobuf-encoded ExportTraceServiceRequest message.
> 
> The default URL path for requests that carry metric data is /v1/metrics and the request body is a Protobuf-encoded ExportMetricsServiceRequest message.
> 
> The default URL path for requests that carry log data is /v1/logs and the request body is a Protobuf-encoded ExportLogsServiceRequest message.
> 
> **The client MAY gzip the content and in that case MUST include “Content-Encoding: gzip” request header. The client MAY include “Accept-Encoding: gzip” request header if it can receive gzip-encoded responses.**
> 
> Non-default URL paths for requests MAY be configured on the client and server sides.
> 
> https://opentelemetry.io/docs/specs/otlp/#otlphttp

Shall we reject the `Accept-Encoding: gzip` when validating schema? Currently, it is not supported.

* If we reject it, this should be considered as a fix, and a new changelog entry is required.
* If not:
    * Do nothing, and customers might mis-configure it.
    * Implement it, out of the scope of this project, and a new changelog entry is required (fix or feature?).

WDYT? We might need PM's help.

Reply:
definitely something to be handled separately to this PR, but good catch. I'm creating an item right now: KAG-4871

Reply:
makes sense